src/main/java/io/cryostat/net/web/http/AbstractAuthenticatedRequestHandler.java

```diff
@@ -104,7 +108,7 @@ public void handle(RoutingContext ctx) {
             throw e;
         } catch (Exception e) {
             if (isAuthFailure(e)) {
-                throw new HttpException(401, "HTTP Authorization Failure", e);
+                throw new HttpException(403, "HTTP Authorization Failure", e);
```

Comment:
Are the 401 exceptions supposed to be wrapped?

Reply:
Not sure what you mean?

Reply:
Oh, sorry, I re-read the code again and there was a part that was not expanded within the GitHub code view, and I got confused. Looks fine to me.